mlflow/metrics/genai/genai_metric.py

```diff
@@ -599,3 +605,94 @@ def eval_fn(
         metric_metadata=metric_metadata,
         genai_metric_args=genai_metric_args,
     )
+
+
+def _filter_by_field(df, field_name, value):
+    return df[df[field_name] == value]
```

Comment:
/ai Is `df[df[field_name] == value]` is equivalent to `df.loc[df[field_name] == value]`?